src/main/python/analyzers/EmotionAnalyzer.py

```diff
@@ -0,0 +1,27 @@
+import pandas as pd
+from nrclex import NRCLex
+
+
+def getEmotions(text):
+    """
+    Get emotion affect frequencies for the input text
+
+    :param text: the input text
+    :return: a dict with the following emtoins: fear, anger, trust, surprise, sadness, disgust, joy, anticipation
+    """
+    textObject = NRCLex(text)
+    emtionsDict = textObject.affect_frequencies
+    if 'anticipation' not in emtionsDict:
+        emtionsDict['anticipation'] = 0.0
+    [emtionsDict.pop(key) for key in ['positive', 'negative', 'anticip']]
```

Comment:
Why are you removing these keys from the dict? Do we not want these columns?

Reply:
'positive/negative' emotion values should be provided by the sentimentanalyzer, so i think it will be redundant to return those values by the emotionanalyzer as well. the 'anticip' value is always 0, however, the 'anticipation' value does have values other than 0, but it is only returned if the tweet has words that convey the feeling of anticipation. i reported an issue to the package owners describing this behaviour. for now, i'm adding anticipation=0 for tweets that don't have 'anticipation' words in them

Reply:
Ah okay, I see. So this ne library performs sentiment analysis and the emotion behind the text? That's pretty cool. I agree with what you did here, unless there's a way to only run the emotions and not the sentiment too. It might save computation time if you can ignore those right off the bat instead of removing them after. If not, it's not a big deal. We can only do what the library allows!

Reply:
unfortunately there isn't. NRCLex(text) returns an NRCLex object, and emotion frequency values are contained in the '.affect_frequencies' attribute

Reply:
![Screenshot from 2020-11-11 20-55-53](https://user-images.githubusercontent.com/7900567/98885775-a0e8be00-2460-11eb-971c-773ee6c49399.png)

Reply:
Very nice... it almost seems pointless to have a separate file for handling semantics if this does it already... I guess it's a different tool for judging the semantics though. What do you think?

Reply:
are you talking about the azure based analyzer? if that's what you mean, then yes, practically speaking, it is pointless to include both. However, NRCLex is only a lexicon, so its very basic honestly, and you can always improve on it. i think the azure models are AI based, which makes them fancier i guess

Reply:
Yeah, that's what I meant. Alright, sounds like it's good to keep using Azure then. I'll get your code merged in now.